### PR TITLE
Add udev rule for Zenfone 8

### PR DESCRIPTION
**Pull request by J-Ahlers**
These rules are needed to see devices in sideload mode after calling "adb -d reboot sideload" with the phone attached.

Tested on my Zenfone 8.

### CODE REVIEW

**Comment by M0Rf30** [comment]
Hi and thank for your contribution.
Seems weird that the same product id is used for more than one mode.
@JoesCat what do you think about this?

**Comment by JoesCat** [comment]
Back in the past, everything that connected to a PC seemed to be drivers, drivers, drivers, but this is also an Achilles heel for Windows security since a lot of viruses want to dig-in below the OS level.
Additionally, with the fact that black hats now also hold the keys to generate their own certificates, I suspect Microsoft is going to be more insistent on they be the ones to provide the drivers (sort of like how linux distros compile their own drivers).
As a manufacturer of a device, I think you'd want to K.I.S. on what a device driver is going to be so that it is K.I.S. on what microsoft provides as a device driver - and provide all the fancy features at the user software level.

From our point of view, I suspect we're going to see a lot more overlapping device IDs.
Samsung seems to be doing this for a long time now where their mtp and adbmtp device IDs overlap. Other device manufacturers also appear to re-use IDs.

We need a better method of distinguishing the android modes, and this might not exist at the 51-android.rules level, but more likely at the user software level, such as adb.

...until something better comes along, I think that, from the point of view of 51-android.rules, you can still think of it as.... 99.9% of users are plain users who simply want to connect their phone to their PC and grab some files, and for the 0.1% of users who are actually developing and experimenting with their phones, they will know better what they need to modify to get 51-android-rules to do what it needs to do.

Right now, the patch is:
```
#   Zenfone 8
ATTR{idProduct}=="4dae", GOTO="adb"
ATTR{idProduct}=="4dae", GOTO="adbfast"
ATTR{idProduct}=="4dae", GOTO="adbmtp"
GOTO="android_usb_rules_end"
LABEL="not_Asus"
```

To satisfy the 99.9% of common users, I'd suggest going with:
```
#   Zenfone 8 (4dae=adb 4dae=mass_storage,adb 4dae=fastboot,adb)
ATTR{idProduct}=="4dae", GOTO="adbmtp"
ATTR{idProduct}=="4dae", GOTO="adbfast"
GOTO="android_usb_rules_end"
LABEL="not_Asus"
```
...and for the 0.1% of users who are developing their device, they would know how to comment-out the line with adbmtp, which 99.9% of common users would have no clue about anyways.
There's no need to have a seperate line for adb, since it's taken care of by the line containing adbmtp. The line containing adbfast never executes under normal circumstances because the line with adbmtp is executed first, however, the 0.1% of developers who know enough how to flash a device should also know enough how to add a "#" on the line with adbmtp for the times they actually plan to flash their device.

In summary, for 99.9% of users, it runs as GOTO adbmtp

**Comment by JoesCat** [comment]
Hi @J-Ahlers, would you have some device ID values for the phone in non-android mode? It would be helpful to add these to the comments - Thanks!

**Comment by M0Rf30** [comment]
I'm closing this. Will be re-opened on if it's already required
